fetch_contributors.py

```diff
@@ -0,0 +1,44 @@
+"""This module writes every contributor to contributor.md"""
+
+import urllib.request
+import os
+import json
+
+#List of contributor object (name,weburl)
+CONTRIBUTORS = []
+
+#first page with 100 contributors
+with urllib.request.urlopen("https://api.github.com/repos/hacktoberfest17/" +
```

Comment:
make sure we get all the contributors, by default github only shows 100.

Reply:
I did this with the pagination